### PR TITLE
OBDS-129 Fix importing stateProvince in locations and add missing location group data in locations.csv

### PR DESCRIPTION
**Pull request by hdsldv**

### CODE REVIEW

File: grails-app/services/org/pih/warehouse/data/LoadDataService.groovy [hunk 1]
```diff
@@ -33,7 +33,17 @@ class LoadDataService {
...
 
         ImportDataCommand command = new ImportDataCommand();
-        command.setData(csvReader.readAll());
+
+        List<Map<String, String>> csvItems = csvReader.readAll()
+
+        for (i in 0..< csvItems.size()) {
+            Map<String, String> currentItem = csvItems.get(i);
+
+            currentItem.put("stateOrProvince", currentItem.get("stateProvince"));
```
Comment (awalkowiak) [inline grails-app/services/org/pih/warehouse/data/LoadDataService.groovy:42]: I think it's easier to rename the column name in file from `stateProvince` to `stateOrProvince` to match the field on address.

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: grails-app/services/org/pih/warehouse/data/LoadDataService.groovy (hunk 1 above)
Comment (hdsldv) [reply grails-app/services/org/pih/warehouse/data/LoadDataService.groovy]: done

File: grails-app/services/org/pih/warehouse/data/LoadDataService.groovy [hunk 2]
```diff
@@ -33,7 +33,7 @@ class LoadDataService {
         CSVMapReader csvReader = new CSVMapReader(csvURL.newInputStream().newReader());
 
         ImportDataCommand command = new ImportDataCommand();
-        command.setData(csvReader.readAll());
+        command.setData(command.setData(csvReader.readAll()));
```
Comment (awalkowiak) [inline grails-app/services/org/pih/warehouse/data/LoadDataService.groovy:36]: Doubled `command.setData`

**Review by awalkowiak — CHANGES_REQUESTED** [review]

File: grails-app/services/org/pih/warehouse/data/LoadDataService.groovy (hunk 2 above)
Comment (hdsldv) [reply grails-app/services/org/pih/warehouse/data/LoadDataService.groovy]: fixed

**Pull request merged**